UnitySDK/Assets/ML-Agents/Scripts/Agent.cs

```diff
@@ -1083,5 +1091,23 @@ public static void ObservationToTexture(Camera obsCamera, int width, int height,
             RenderTexture.active = prevActiveRT;
             RenderTexture.ReleaseTemporary(tempRT);
         }
-    }
+
+        /// <summary>
+        /// Sets the custom output for the agent for this episode.
+        /// </summary>
+        /// <param name="customOutput">New value of the agent's custom output.</param>
+        public void SetCustomOutput(CustomOutput customOutput)
+        {
+            info.customOutput = customOutput;
+        }
+
+        /// <summary>
+        /// Creates a new instance of a custom output object.
+        /// </summary>
+        /// <returns>A reference to a new CustomOutput object.</returns>
+        static public CustomOutput CreateCustomOutput()
```

Comment:
I think it is fine to let the developer instantiate his own CustomOutput. This method feels redundant

Reply:
My thinking was that could be problematic since the user has to then know about the `CommunicatorObjects` namespace, where the `CustomOutput` class lives. If the C# namespace of the custom_output.proto file ever changes due to some kind of internal refactoring, users would have to adopt their code.

But sure, I'll make this change.